web/src/lib/domain-logic/switchCurrentBusiness.ts

```diff
@@ -0,0 +1,12 @@
+import { UserData } from "@businessnjgovnavigator/shared/userData";
+
+export const switchCurrentBusiness = (userData: UserData, newCurrentBusinessId: string): UserData => {
+  if (!Object.keys(userData.businesses).includes(newCurrentBusinessId)) {
+    throw "Business does not exist";
```

Comment:
[dust] Should there be an error object that gets created or a console log? What happens when you throw a string?

Reply:
This is a good catch.

Really, it doesn't matter at all because the only reason to throw here is to prevent a situation that should never occur in the first place. This throw should never actually get thrown or have to be caught.

And technically you can throw anything you want in JS, including a string or even nothing at all. It throws just the same. But you're right, it's considered good form to throw an `Error` object instead of just a raw string.